reference/configuration/monolog.rst

```diff
@@ -86,7 +91,13 @@ Full Default Configuration
                 http://symfony.com/schema/dic/monolog
                 http://symfony.com/schema/dic/monolog/monolog-1.0.xsd">
 
-            <monolog:config>
+            <!-- By default, use-microseconds is set to true
+                 including 'use-microseconds="false' forces
+                 the logger to reduce the precision in the
+                 datetime field of the log messages from
+                 microsecond to second. Avoiding a call to the
+                 microtime(true) function and the subsequent parsing. -->
+            <monolog:config use-microseconds="true">
```

Comment:
No need to put the comments in XML also. We typically just put the comments in YAML - it's the most visible and avoids duplication :)